docs/csharp/how-to/search-strings.md

```diff
@@ -0,0 +1,61 @@
+---
+title: "How to: Search Strings (C# Guide)"
+ms.date: 02/21/2018
+ms.prod: .net
+ms.technology: 
+  - "devlang-csharp"
+ms.topic: "article"
+helpviewer_keywords: 
+  - "searching strings [C#]"
+  - "strings [C#], searching with String methods"
+ms.assetid: fb1d9a6d-598d-4a35-bd5f-b86012edcb2b
+ms.author: "wiwagn"
+---
+
+# How to: Search Strings Using String Methods (C# Programming Guide)
+
+You have two main strategies that you can use to search for text in strings. Member methods of the <xref:System.String> class search for specific text. Regular expressions search for patterns in text.
+
+The [string](../language-reference/keywords/string.md) type, which is an alias for the <xref:System.String?displayProperty=nameWithType> class, provides a number of useful methods for searching the contents of a string. Among them are <xref:System.String.Contains%2A>, <xref:System.String.StartsWith%2A>, <xref:System.String.EndsWith%2A>, <xref:System.String.IndexOf%2A>, <xref:System.String.LastIndexOf%2A>. The <xref:System.Text.RegularExpressions.Regex?displayProperty=nameWithType> class provides a rich vocabulary to search for patterns in text. In this article, you learn these techniques and how to choose the best method for your needs.
+
+## Does a string contain text?
+
+The <xref:System.String.Contains%2A?displayProperty=nameWithType>, <xref:System.String.StartsWith%2A?displayProperty=nameWithType> and <xref:System.String.EndsWith%2A?displayProperty=nameWithType> methods search a string for specific text. The following example shows each of these methods, and a variation that uses a case insensitive search:
+
+[!code-csharp-interactive[search strings using methods](../../../samples/snippets/csharp/how-to/strings/SearchStrings.cs#1)]
+
+The preceding example demonstrates an important point for using these methods. Searches are **case-sensitive** by default. You use the <xref:System.StringComparison.CurrentCultureIgnoreCase> enum value to specify a case insensitive search. All these methods default to case-sensitive search.
```

Comment:
There is a slight duplication in this paragraph: 
>Searches are **case-sensitive** by default.

and
>All these methods default to case-sensitive search.

are separated just by one sentence. Was it intentional?